tests/interned.rs

```diff
@@ -12,6 +12,14 @@ impl salsa::Database for Database {
     }
 }
 
+impl salsa::ParallelDatabase for Database {
```

Comment:
This doesn't work without a patch, because rustc thinks that `Database` contains an `Runtime<Database>`, which has `RefCell`